test/browser/support/setup.js

```diff
@@ -10,7 +10,7 @@ BeforeAll({ timeout: 2 * 5000 }, async function () {
         // Not headless so we can watch test runs
         headless: false,
         // Slow so we can see things happening
-        slowMo: 1000,
+        //slowMo: 1000,
```

Comment:
It's best to leave the slowMo line commented out, otherwise it will take a long time to run through the screens in CI/CD environments, especially when more screens are added.